service_listener.go

```diff
@@ -226,21 +226,21 @@ func getIngressBasedDomainServiceMap(result map[string]rule, c *client.Client, l
 
 	ingresses, err := c.Ingress(api.NamespaceAll).List(listOptions)
 	if err != nil {
-		glog.Fatalf("Failed to list ingress: %v", err)
+		glog.Infof("Failed to list ingress: %v", err)
```

Comment:
Great - this was causing the crash loop in my cluster since I didn't have an ingress deployed.